src/console_cmds.cpp

```diff
@@ -1112,15 +1112,14 @@ DEF_CONSOLE_CMD(ConExec)
 
 	if (argc < 2) return false;
 
-	FILE *script_file = FioFOpenFile(argv[1], "r", BASE_DIR);
+	auto script_file = FioFOpenFile(argv[1], "r", BASE_DIR);
 
-	if (script_file == nullptr) {
+	if (!script_file.has_value()) {
```

Comment:
This PR is inconsistent whether to use `x` or `x.has_value()`. I'm seeing both constructs being used. Is that deliberate?

Reply:
`nullptr` is definitely wrong, I intended to use `has_value()` consistently, but clearly I failed in places.